kvbc/src/v4blockchain/detail/latest_keys.cpp

```diff
@@ -27,10 +27,12 @@ auto getSliceArray(const Sliceable&... sls) {
 }
 
 LatestKeys::LatestKeys(const std::shared_ptr<concord::storage::rocksdb::NativeClient>& native_client,
-                       const std::optional<std::map<std::string, categorization::CATEGORY_TYPE>>& categories)
-    : native_client_{native_client}, category_mapping_(native_client, categories) {
+                       const std::optional<std::map<std::string, categorization::CATEGORY_TYPE>>& categories,
+                       std::function<BlockId()>&& f)
+    : native_client_{native_client}, category_mapping_(native_client, categories), comp_filter_(std::move(f)) {
   if (native_client_->createColumnFamilyIfNotExisting(v4blockchain::detail::LATEST_KEYS_CF,
-                                                      concord::storage::rocksdb::getLexicographic64TsComparator())) {
+                                                      concord::storage::rocksdb::getLexicographic64TsComparator(),
+                                                      getCompFilter())) {
     LOG_INFO(V4_BLOCK_LOG,
```

Comment:
Should convert all LOG_INFO to LOG_TRACE

Reply:
those are one time prints, It's nice to have them

Reply:
thanks @cloudnoize